taskcoordinator/app/src/main/resources/application.properties

```diff
@@ -6,10 +6,13 @@ spring.datasource.password=123
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=validate
 spring.datasource.initialization-mode=always
+
 spring.jpa.properties.hibernate.default_schema=${database.schema.name}
 spring.flyway.schemas=${database.schema.name}
```

Comment:
what is the difference between this properties? I mean... I understand but why the first one is not ehat we need?